semver/can_parse.ts

```diff
@@ -2,9 +2,13 @@
 import { SemVer } from "./types.ts";
 import { parse } from "./parse.ts";
 
+/**
+ * @deprecated (will be removed in 0.211.0) use string as an argument
+ */
+export function canParse(version: SemVer): boolean;
```

Comment:
Looks like we also need `export function canParse(version: string): boolean;` line?

Reply:
added